docs/tutorials/multinode/terraform/htcondor/autoscaler.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script for resizing managed instance group (MIG) cluster size based
+# on the number of jobs in the Condor Queue.
```

Comment:
I am assuming that this is some standard file and am not going to review.  Let me know if I should actually review for python readability, etc.

Reply:
@jrossthomson if this is a standard file, could you include a link in the comment here to its documentation?

If it's not a standard file we should have testing for it, especially since a malfunction here could cause unintended billing for users.